### ISSUE TITLE
KSP 1.12.1/MJ#1092: PVG Ascent Guidance failure/exception

### ISSUE THREAD

**Issue by thunder175** [MuMech/MechJeb2#1474]
When launching a rocket using Bluedog Design Bureau parts on KSP 1.12.1 using MechJeb Dev version 1092 using PVG guidance. Upon staging I received an error exception, and the ascent guidance GUI went blank. MJ was locked to the last guidance instructions with no way to shut down the autopilot. The logs were filled with exception errors being thrown constantly  until I returned to the KSC. I then went into the tracking station, reentered the flight, and MJ returned to normal awaiting ascent guidance inputs from the user. 

[Player.zip](https://github.com/MuMech/MechJeb2/files/6824962/Player.zip)
[[Pathfinder] Fenris-Alpha.zip](https://github.com/MuMech/MechJeb2/files/6824975/Pathfinder.Fenris-Alpha.zip)
[mods.txt](https://github.com/MuMech/MechJeb2/files/6824979/mods.txt)

**Comment by lamont-granquist** [comment]
It looks like this is due to upgrading the MechJeb.dll without getting the new System.Buffers.dll which is shipped in the Plugin directory:

```
% unzip -l MechJeb2-2.12.0.0-1093.zip 
Archive:  MechJeb2-2.12.0.0-1093.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  07-26-2021 09:33   MechJeb2/
[...]
        0  07-26-2021 09:34   MechJeb2/Plugins/
  4333568  07-26-2021 09:34   MechJeb2/Plugins/MechJeb2.dll
    20856  07-26-2021 09:33   MechJeb2/Plugins/System.Buffers.dll
---------                     -------
 11153405                     61 files
```

I've been trying to get a repo for ages and its been perfectly solid, but I just deleted that file and it repro'd immediately for me.

**Comment by lamont-granquist** [comment]
nevermind, i got a repro with System.Buffers.dll there so it looks like that was just a coincidence.

(but now i have a repro)

**Comment by lamont-granquist** [comment]
fixed by  #1481
